code/modules/clothing/head/helmet.dm

```diff
@@ -340,3 +340,42 @@
 	item_state = "helmet"
 	species_fit = list(INSECT_SHAPED)
 	body_parts_covered = FULL_HEAD|MASKHEADHAIR
+
+/obj/item/clothing/head/helmet/xcom
```

Comment:
new ablino meta? this has flashbang dampening unlike the EOD

Reply:
Huh, weird. I would have assumed the EOD would also be under the "helmet" umbrella, but I guess it's not.

Even so, this is only as good or bad as a regular sec helmet statwise, and it requires some research. I think it's fine.